src/main/java/de/ait/secondlife/controllers/OfferController.java

```diff
@@ -406,6 +412,57 @@ public ResponseEntity<OfferResponseWithPaginationDto> search(
         return ResponseEntity.ok(service.searchOffers(getPageable(page, size, sortBy, isAsc), locationId, pattern));
     }
 
+    @GetMapping("/user/{id}/bid")
+    @Operation(
+            summary = "Get all offers by user id in which the user has placed a bid",
+            description = "Receiving all offers by user id available in the database with pagination" +
+                    " in which the user has placed and the offers have statuses : " +
+                    "AUCTION_STARTED, QUALIFICATION, COMPLETED, CANCELED, BLOCKED_BY_ADMIN"
+    )
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "Successful operation",
+                    content = @Content(mediaType = "application/json", schema = @Schema(implementation = OfferResponseWithPaginationDto.class))),
+    })
+    public ResponseEntity<OfferResponseWithPaginationDto> getAllByUserBidByUserId(
```

Comment:
1) is this for some other task? not for https://cohort38.atlassian.net/browse/SL-148 ?
2) names of methods in the controller, service, and repository are too complicated. could you write analogue in russian? I think we can simplify it